astroquery/mast/cutouts.py

```diff
@@ -414,6 +431,10 @@ def get_cutouts(self, *, coordinates=None, size=5, product='SPOC', sector=None,
         response : A list of `~astropy.io.fits.HDUList` objects.
         """
 
+        if timeout:
+            log.info(f'Request timeout upper limit is being changed to {timeout} seconds.', InputWarning)
+            self._service_api_connection.TIMEOUT = timeout
```

Comment:
That specifying a function parameter permanently changes the value of a constant is very unexpected. It would be better to make the timeout configurable using the `astropy` configuration system, see https://docs.astropy.org/en/stable/config/index.html#usage-tips